cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -361,29 +315,6 @@ func getControllerManagerCommand(cfg *kubeadmapi.ClusterConfiguration) []string
 		defaultArguments["cluster-name"] = cfg.ClusterName
 	}
 
-	// TODO: The following code should be remvoved after dual-stack is GA.
-	// Note: The user still retains the ability to explicitly set feature-gates and that value will overwrite this base value.
-	enabled, present := cfg.FeatureGates[features.IPv6DualStack]
-	if present {
-		defaultArguments["feature-gates"] = fmt.Sprintf("%s=%t", features.IPv6DualStack, enabled)
```

Comment:
does this makes sense?
we already have the config with the feature gate enabled

Reply:
this passes the IPv6DualStack feature gate to the KCM by default, so i'd say it's needed.
it is passed to the API server and other components too:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/controlplane/manifests.go#L197

Reply:
Ahh, I got it now, let me change it